pantavisor-run.in

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
what is the purpose of this script? shouldn't this be in appengine/ or scripts/?

Reply:
this is a wrapper script that "normalizes" the mount namespace so that it has certain legacy directories etc. before starting pantavisor.

This is getting run in a namespace environment that hides these things from host (like a bindmount to /storage from /var/pantavisor/storage etc.)

Not entirely sure about this one, but i needed a place to make those legacy directories available. In embedded mode we have the "old" file system directories still - once we try to move away from those in embedded we might need this script there as well. Hence, I put it in top level directory, but if you have a better place let me know.

Reply:
IMO it's ok if we keep this in this repo, but I'd move it to the scripts/ dir (not pv-scripts but scripts in this repo)  or maybe even to a new scripts/appengine dir if we expect other appengine scripts in the future.